mozilla-release/browser/components/preferences/in-content/privacy.xul

```diff
@@ -170,16 +138,12 @@
               preference="pref.privacy.disable_button.change_blocklist"/>
     </hbox>
   </vbox>
-<<<<<<< HEAD
-#endif
-||||||| merged common ancestors
-=======
   <vbox>
```

Comment:
It seems, we don't want to hide this block (it was refactored from `checkbox id="privacyDoNotTrackCheckbox"` above and moved down here). It's related to DoNotTrack header preference, which we still keep.